measures/BuildExistingModel/measure.rb

```diff
@@ -600,8 +600,14 @@ def run(model, runner, user_arguments)
       register_value(runner, 'weight', weight.to_s)
     end
 
-    if args['sample_weight'].is_initialized
-      register_value(runner, 'weight', args['sample_weight'].get.to_s)
+    sample_weight = args['sample_weight'].is_initialized ? args['sample_weight'].get : nil
+    if bldg_data.keys.include?('sample_weight')
+      sample_weight = bldg_data['sample_weight']
+      register_value(runner, 'sample_weight', sample_weight.to_s)
+    end
+
+    if !sample_weight.nil?
+      register_value(runner, 'weight', sample_weight.to_s)
```

Comment:
Why are these registered as different variables? Should it just be one or the other gets stored? I do see there is a "weight" and a "sample_weight" in results.json, but then just "sample_weight" is output in results_up00.csv, can you help clarify this?

Reply:
I think it gets registered so that it will show up in the outputs in results.csv. Then we can use it in downstream postprocessing.

Reply:
But doesn't it [get removed](https://github.com/NREL/buildstockbatch/blob/develop/buildstockbatch/postprocessing.py#L168) in favor of using `sample_weight`?

Reply:
It would appear that it does.

Reply:
Any recommendations what should be done here in regard to `weight`? Nothing? Or should I try removing it and see what happens?

Reply:
I'm a little lost on where `sample_weight` gets used downstream, because it seems like `sample_weight` never got set here before, only `weight`

Reply:
`sample_weight` is a measure argument for BuildExistingModel. As long as this argument gets assigned a value, it will "register" as `build_existing_model.sample_weight` (i.e., you don't need the `register_value(runner, 'sample_weight', ...)` call). This PR "re-registers" (overwrites) the `sample_weight` value as it exists in the buildstock.csv.